assets/src/components/stopCard.tsx

```diff
@@ -73,6 +73,13 @@ const StopCard = ({
       autoPanPadding={autoPanPadding?.padding || [20, 20]}
       autoPanPaddingTopLeft={autoPanPadding?.paddingTopLeft}
       autoPanPaddingBottomRight={autoPanPadding?.paddingBottomRight}
+      // Leaflet implements Popup as a singleton, which means that the
+      // autoPanPadding settings won't be updated while the map is
+      // in-memory. Setting a key that has a stringified version of
```

Comment:
```suggestion
      // autoPanPadding setting won't be updated because it is only
      // applied when the `<Popup/>` is created.
      // Setting a key that has a stringified version of
```